src/entry_points/eest_utils.py

```diff
@@ -0,0 +1,87 @@
+"""
+CLI interface for common EEST shortcuts and utilities.
+"""
+
+import os
+import shutil
+import subprocess
+
+import click
+
+from logger import setup_logger
+
+logger = setup_logger(__name__)
+
+
+@click.group()
+def eest():
+    """
+    EEST commands for common repo utilities.
+    """
+    pass
+
+
+@eest.command()
+def init():
+    """
+    Initializes and installs all required packages for the repository:
+    ```
+    python3 -m venv ./venv/
+    source ./venv/bin/activate
+    pip install -e '.[docs,lint,test]'
+    ```
+    """
+    logger.info("Creating a virtual environment: `python3 -m venv ./venv/`")
```

Comment:
Example command line in this string should reflect the actual comment running in the next line (maybe simply create `args = [...` and use it twice).

Reply:
[5652f31](https://github.com/ethereum/execution-spec-tests/pull/461/commits/5652f3159b6411c71c82b4ce34329daa1ec2403d)